text/0069-api-v2.md

```diff
@@ -105,29 +108,20 @@ To enable API V2, which also enables the keyspace API:
 To manage keyspace:
 
 ```bash
->> config keyspaces show
->> config keyspaces create <keyspace name>
->> config keyspaces delete <keyspace name>
-
-# example: config keyspaces set-property foo default-config.raw-client.ttl-secs 100000
->> config keyspaces set-property <keyspace name> <property-path> <property-value>
+>> config keyspaces show [--deleted]
+[
+  {
+     name: string,
+     id: int64,
+     created_at: timestamp
+  }
+]
 
->> config keyspaces delete-property <keyspace name> <property-path>
->> config keyspaces flashback <new keyspace name>
+>> config keyspaces create <keyspace name> [--id <Keyspace ID>]
```

Comment:
Is there any restriction on the provided `Keyspace ID` ? For example, what if the ID is the same as another Keyspace ?

Reply:
updated

Reply:
What if two concurrent create requests with different name but same id ?

AFAIK etcd provides [transaction API](https://etcd.io/docs/v3.5/learning/api/#transaction) similar to CAS. It seems to be difficult to solve such a race condition.